admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.js

```diff
@@ -184,6 +189,39 @@ export default class CombinationsManager {
     });
   }
 
+  /**
+   * @param {HTMLElement} button
+   *
+   * @private
+   */
+  async removeCombination(button) {
+    try {
+      const $deleteButton = $(button);
+      const modal = new ConfirmModal(
+        {
+          id: 'modal-confirm-delete-combination',
```

Comment:
Please move selector in map file

Reply:
It's not really needed because it's never used for selection The id used here is only temporary while we create the confirm modal It's been used this way everywhere else I'm not sure it brings much value 🤔